packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -40,56 +41,193 @@ const LOADPANEL_MESSAGE_CLASS = 'dx-loadpanel-message';
 const MESSAGE_SELECTOR = '.' + LOADPANEL_MESSAGE_CLASS;
 const LOADPANEL_CONTENT_CLASS = 'dx-loadpanel-content';
 const LOADPANEL_PANE_HIDDEN_CLASS = 'dx-loadpanel-pane-hidden';
+const LOADINDICATOR_CLASS = '.dx-loadpanel-indicator';
+
+QUnit.module('accessibility', () => {
+    QUnit.module('message option tests', () => {
+        ['', undefined, null].forEach(state => {
+            const stateString = (state === '') ? 'empty' : state;
+            QUnit.test(`message is set to ${stateString} on init`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                const loadPanelMessage = loadPanel.$wrapper().find('.dx-loadpanel-message').text();
 
-QUnit.module('init', {
-    beforeEach: function() {
-        fx.off = true;
-    },
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
+            });
 
-    afterEach: function() {
-        fx.off = false;
-    }
-}, () => {
-    QUnit.test('rendered markup', function(assert) {
-        const $element = $('#loadPanel').dxLoadPanel({ message: 'Test Loading Message', visible: true }); const $content = $element.dxLoadPanel('instance').$content();
+            QUnit.test(`message is set from ${stateString} to non empty on runtime`, function(assert) {
+                const loadPanel = $('#loadPanel').dxLoadPanel({
+                    visible: true,
+                    message: state,
+                    showIndicator: false
+                }).dxLoadPanel('instance');
+                let loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
 
-        assert.ok($element.hasClass(LOADPANEL_CLASS));
-        assert.ok($content.hasClass(LOADPANEL_CONTENT_CLASS), 'Load Indicator created');
-        assert.ok($content.find(MESSAGE_SELECTOR).length);
-        assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
```

Comment:
I'm afraid these tests are still not what i meant :(
I mean writing tests to check aria-attributes after message option runtime change, testing [this your code row](https://github.com/DevExpress/DevExtreme/pull/27080/files#diff-53b011e4c36d90ad74e975c0d3719f7a2a69b4893caec03a31a2086859fed5bbR135). We don't need to test everything here, only accessibility. 

For example here is how the first test can look like:
```javascript
QUnit.module('accessibility', () => {
    QUnit.module('aria-label dependent on message option', () => {
        ['', undefined, null].forEach(emptyValue => {
            QUnit.test(`equals default loading text if message equals to ${emptyValue} on init`, function(assert) {
                const loadPanel = $('#loadPanel').dxLoadPanel({
                    visible: true,
                    message: emptyValue,
                    showIndicator: false
                }).dxLoadPanel('instance');

                const $loadPanelWrapper = loadPanel.$wrapper();
                assert.strictEqual($loadPanelWrapper.attr('aria-label'), 'Loading...', 'wrapper aria-label is correct when message is empty on init');
            });
```

Reply:
[bb57796](https://github.com/DevExpress/DevExtreme/pull/27080/commits/bb57796b0f4e0953f491b0adee68e1f35762a715)